docs/markdown/Syntax.md

```diff
@@ -695,7 +695,7 @@ additive_expression: multiplicative_expression | (additive_expression additive_o
 additive_operator: "+" | "-"
 argument_list: positional_arguments ["," keyword_arguments] | keyword_arguments
 array_literal: "[" [expression_list] "]"
-assignment_expression: conditional_expression | (logical_or_expression assignment_operator assignment_expression)
+assignment_statement: conditional_expression | (logical_or_expression assignment_operator assignment_expression)
```

Comment:
I think this should be `assignment_statement: IDENTIFIER assignment_operator expression`

Reply:
Actually, meson++ has this https://github.com/dcbaker/meson-plus-plus/blob/b7099bd3e8b3d1a2a7c8fea63821993ccb7da7ff/src/frontend/parser.yy#L106,
so I think this should be
`assignment_statement: expression asssignment_operator expression`